src/utils/api.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { Cookie } from '../services/cookie';
+import { TOKEN_URL, MAIN_URL } from '../config/constants';
+
+const token = Cookie.getToken();
+
+if (token === undefined) {
```

Comment:
condition can be simplified

Reply:
Fixed like that: `!token && getAccessTokenFromAPI();`